navie/extract_changes.py

```diff
@@ -22,6 +22,9 @@ def __eq__(self, other):
 
 
 def extract_changes(content: str) -> List[FileUpdate]:
+    # Gemini tends to use ``` instead of closing CDATA properly.
+    content = content.replace("```</", "]]></")
```

Comment:
Do you need to see this at the beginning of the line?

Reply:
That's what I've seen so far but CDATA doesn't need to close at the beginning of the line. (MD fence does, so maybe Gemini wouldn't generate a fence unless it sits at the start of line...)

Do you think it's too aggressive?

Reply:
I think, since fences have to start a line; the pattern can start at the beginning of a line. 

But in practice it probably doesn’t matter much because it will be generated at the beginning of the line anyway.

Reply:
It's not a correct fence anyway (they also have to be followed by newline or, if opening, the language spec); Gemini just gets delimiters mixed up, so who knows what else it can do in the confusion 🤷🏼‍♂️